src/CarbonAware.WebApi/src/Program.cs

```diff
@@ -51,7 +70,13 @@
 if(!successfulEmissionServices)
```

Comment:
Is it possible to log the error message when the exception is caught or do we not have the handle to logger instance at this point?

Reply:
The logger is dependent on the `app = buider.Build()`  which happens on line 70, so we have to run that before we can log the exception